os/arch/arm/Kconfig

```diff
@@ -25,12 +25,26 @@ config ARCH_CHIP_S5J
 	select ARMV7R_MEMINIT
 	---help---
 		Samsung IoT SoC architectures (ARM Cortex R)
+
+config ARCH_CHIP_BCM4390X
+	bool "CY 4390X series"
+	select ARCH_CORTEXR4
+	select ARCH_HAVE_MPU
+	select ARMV7M_MPU
+	select ARMV7M_MPU_NREGIONS
```

Comment:
The above options takes a numeric value, is it ok to just select it.  I assume we need to mention number of regions in this SOC, may be 8 to 12. Please review these options. In select we only try to enable options which have 0 or 1 values

Reply:
it will be removed.  these definitions are used in defconfig file.

BR, HJ Lee.